.github/workflows/tests.yml

```diff
@@ -66,7 +66,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        symfony: ['2', '3', '4']
+        symfony: ['2', '3', '4', '5', '6']
```

Comment:
This alone won't work. You'll need to exclude symfony 5 on php 7.1, and symfony 6 on all php 7.x versions

Reply:
It also won't work because no such version of `dunglas/symfony-lock` exists.

Reply:
oops, sorry jeroeny.

@GrahamCampbell my thought would be to drop support for symfony 2 and 3 (those are super old, and people stuck on those versions can still use the existing releases of symfony) and change the build to use flex to lock the symfony version instead of dunglas/symfony-lock. do you agree?

Reply:
sounds good.

Reply:
is the environment variable `SYMFONY_REQUIRE` used to lock the Symfony version in Flex? See new commit

Reply:
thanks a lot for also handling the ci setup, jeroeny